Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12607,6 +12609,19 @@ - (CGSize)sizeForLegacyFormControlPickerViews
     return static_cast<id<BETextInput>>(self);
 }
 
+- (void)_logMissingSystemInputDelegateIfNeeded:(const char*)methodName
+{
+    if (_asyncInputDelegate)
+        return;
```

Comment:
Wouldn't you not end up logging here if `_asyncInputDelegate == nil`?

Reply:
> Wouldn't you not end up logging here if `_asyncInputDelegate == nil`?

Hm... the intention is to log only when `! _asyncInputDelegate` (hence the early return if `_asyncInputDelegate` is non-null). Maybe I'm missing something obvious?

Reply:
No, no, not enough caffeine! I was missing the obvious bit 🤣